webpack/move_to_foreman/components/common/table/components/TableBody.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table as PfTable } from 'patternfly-react';
+
+import TableBodyMessage from './TableBodyMessage';
+
+const TableBody = ({
+  columns, rows, message, ...props
+}) => {
+  if (message) {
+    return <TableBodyMessage colSpan={columns.length}>{message}</TableBodyMessage>;
+  }
+
+  return <PfTable.Body rows={rows} rowKey={({ rowIndex }) => rowIndex} {...props} />;
+};
+
+TableBody.propTypes = {
+  columns: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
I think you can use `PropTypes.arrayOf()` and remove the need for these `eslint-disable-line`s.